src/main/resources/db/changelog/changes/02-add-is-deleted-to-books-table.yaml

```diff
@@ -0,0 +1,14 @@
+databaseChangeLog:
+  - changeSet:
+      id: addColumn-isDeleted-to-books-table

```

Comment:
you can include this in your first changeset

Reply:
@TetianaDenchyk  you didn't fix this remark with Liquibase :)

Reply:
Yes, I did )) I don't have second changelog both in my local version and here